### PR TITLE
Bug fix for the issue #5264

### PR DESCRIPTION
**Pull request by madankb**
Code fix for the issue "Amount of favorite points not updated when awarding a favorite point in logging view #5264".

### CODE REVIEW

File: main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java [hunk 1]
```diff
@@ -14,25 +24,15 @@
 import cgeo.geocaching.enumerations.LogType;
 import cgeo.geocaching.enumerations.StatusCode;
 import cgeo.geocaching.loaders.UrlLoader;
+import cgeo.geocaching.models.Geocache;
+import cgeo.geocaching.models.Image;
+import cgeo.geocaching.models.TrackableLog;
```
Comment (SammysHP) [inline main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java]: Sorting of imports messed up.

**Comment by madankb** [comment]
@Sammy I am using Android studio 2.0 version which I believe sort the headers in this order. Should I need to rearrange in old order? Do I have any other changes in the code? Please confirm.

**Comment by madankb** [comment]
@SammysHP Is my code fine to merge?.

File: main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java (hunk 1 above)
Comment (SammysHP) [inline main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java]: Sorry, it's ok. I am used to see the current package located at the top and was somehow confused by the diff.

**Comment by SammysHP** [comment]
We do not check the result web page, do we? Instead of just assuming that the favorite point was given to the cache successfully it might be better to check the real result.

As we do the same for other properties of a cache, I think this patch is fine and I'll merge it. Thanks for your contribution!

**Pull request merged**